Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,70 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
+        List<Person> _people = new List<Person>();
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            var interactionManger = new InteractionManager();
+
+        StartOver:
+            interactionManger.PerformInteractions();
+
+            Console.WriteLine("Whould you like to start again?");
+            var answer = Console.ReadLine();
+            if (answer.ToLower() == "y")
+                goto StartOver;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this use of 'goto'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896164)